docs/source/keywords.rst

```diff
@@ -347,91 +347,102 @@ Architectures
 CPU architectures
 -----------------
 
-.. list-table::
-    :widths: 25 65 10
-    :header-rows: 1
-    :align: left
-
-    * -
-      - Description/info
-      - Default
-
-    * * ``Kokkos_ARCH_NATIVE``
-      * Optimize for the local CPU architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_A64FX``
-      * Optimize for ARMv8.2 with SVE Support
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_AMDAVX``
-      * Optimize for AMDAVX architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_ARMV80``
-      * Optimize for ARMV80 architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_ARMV81``
-      * Optimize for ARMV81 architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_ARMV8_THUNDERX``
-      * Optimize for ARMV8_THUNDERX architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_ARMV8_THUNDERX2``
-      * Optimize for the ARMV8_THUNDERX2 architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_BDW``
-      * Optimize for BDW architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_HSW``
-      * Optimize for HSW architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_KNC``
-      * Optimize for KNC architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_KNL``
-      * Optimize for KNL architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_POWER8``
-      * Optimize for POWER8 architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_POWER9``
-      * Optimize for POWER9 architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_SKX``
-      * Optimize for SKX architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_SNB``
-      * Optimize for SNB architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_SPR``
-      * Optimize for Sapphire Rapids architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_ZEN``
-      * Optimize for Zen architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_ZEN2``
-      * Optimize for Zen2 architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_ZEN3``
-      * Optimize for Zen3 architecture
-      * ``OFF``
+Kokkos does not automatically add compiler flags to optimize for a specific CPU architecture.
+When compiling on the machine that the code also runs on, the easiest way to get the CPU code optimized is using ``Kokkos_ARCH_NATIVE=ON`` when configuring.
+In case of compiling for another machine, the CPU architecture can be manually specified. For the available architectures see the following list.
 
+.. flat-table:: CPU architectures
+    :header-rows: 1
+    :stub-columns: 0
+    :widths: 20 20 40 20
+
+    * - Cmake keyword
+      - Architecture/Instruction set
+      - Examples
+      - Vendor
+
+    * - ``Kokkos_ARCH_ZEN3``
```

Comment:
Is there something that can emphasize an horizontal line, or may be just double line to pronounce a bit more the separation between vendors?

Reply:
introduced in https://github.com/kokkos/kokkos-core-wiki/pull/602/commits/a002969daff65f39fdaeebf651d08e0dd88e332e